staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/eventclock/testing/fake.go

```diff
@@ -43,24 +45,54 @@ var _ counter.GoRoutineCounter = (*waitGroupCounter)(nil)
 
 func (wgc *waitGroupCounter) Add(delta int) {
 	if klog.V(7).Enabled() {
-		var pcs [5]uintptr
+		var pcs [10]uintptr
```

Comment:
Can we move the changes related to this logging to a separate PR?

Reply:
+1